client/components/Header.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+class Header extends React.Component{
+render(){
+    return(
+        <div className = 'Header'>
+        <h1><p>Name</p></h1>
```

Comment:
Please mock all data as you would expect to get it from state. 
e.g  {state.firstName}

Reply:
Hi Sis thanks heaps this has been fixed with this commit : https://github.com/kahikatea-2019/MOA/pull/32/commits/9d08c82172833b4f51ba6480270709418fa1d84c